lib/src/kbasesearchengine/events/handler/WorkspaceEventHandler.java

```diff
@@ -644,4 +670,153 @@ private static String toWSRefPath(final List<GUID> objectRefPath) {
         return String.join(";", refpath);
     }
 
+    @Override
+    public StatusEvent updateEvent(final StatusEvent ev)
```

Comment:
I sort of think we should put more constraints re which fields are required in an event per event type and document it better, but see specific comments below